test/metadata.ts

```diff
@@ -95,12 +95,37 @@ describe('metadata', () => {
     const FUNCTION_NAME = 'function-name';
     const FUNCTION_REGION = 'function-region';
 
+    const K_SERVICE = 'k-service';
+    const GOOGLE_CLOUD_REGION = 'google-cloud-region';
+
     beforeEach(() => {
       process.env.FUNCTION_NAME = FUNCTION_NAME;
       process.env.FUNCTION_REGION = FUNCTION_REGION;
+      delete process.env.K_SERVICE;
```

Comment:
Could we use something like sinon to stub instead?  Worried that running the tests will stomp my current session's environment variables.

Reply:
Sinon looked more complicated than needed, but I've updated the code to use before/after hooks to restore the parts of `process.env` I am modifying.